packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -35,6 +35,8 @@ async function handleJSONResponse (res) {
 export default class RequestClient {
   static VERSION = packageJson.version
 
+  #companionHeaders = {}
```

Comment:
We don't need to create a throw away object since that value is overridden in the constructor anyway.
```suggestion
  #companionHeaders
```

Reply:
Thanks. I've implemented your suggestions.